modules/serverless-sinkbinding-kn.adoc

```diff
@@ -0,0 +1,138 @@
+// Module included in the following assemblies:
+//
+// serverless/knstive_eventing/serverless-sinkbinding.adoc
+
+[id="serverless-sinkbinding-kn_{context}"]
+= Using SinkBinding with the Knative CLI (`kn`)
+
+This guide describes the steps required to create, manage and delete a SinkBinding instance using `kn` commands.
+
+// To implement SinkBinding, a Deployment, Job or StatefulSet is required, or alternatively any object which carries a Pod template and can create a Pod.
+
+
+.Prerequisites
+
+* You have Knative Serving and Eventing installed.
+* You have the `kn` CLI installed.
+
+.Procedure
+
+. To check that SinkBinding is set up correctly, create a Knative event display service, or event sink, that dumps incoming messages to its log.
++
+You can create the service by entering the following command:
++
+----
+$ kn service create event-display --image quay.io/openshift-knative/knative-eventing-sources-event-display:v0.13.2
```

Comment:
Should use a specific tag :v0.13.2 for the current release

kn service create event-display --image quay.io/openshift-knative/knative-eventing-sources-event-display:v0.13.2